docs/docs/policies/rules.md

```diff
@@ -256,10 +256,10 @@ spec:
 	rules:
 	    event: security_file_open
 	    filters:
-		- args.pathname=/tmp*
+		- data.pathname=/tmp*
 ```
 
-Arguments can be found on the respective event definition, in this case [security_file_open](https://github.com/aquasecurity/tracee/blob/main/pkg/events/events.goL5293-L529), or the user can test the event output in CLI before defining a policy, e.g:
+Data fields can be found on the respective event definition, in this case [security_file_open](https://github.com/aquasecurity/tracee/blob/main/pkg/events/events.goL5293-L529), or the user can test the event output in CLI before defining a policy, e.g:
```

Comment:
```suggestion
Data fields can be found on the respective event definition, in this case [security_file_open](https://github.com/aquasecurity/tracee/blob/656eb976fbb66aba54c5f306019258e436d4814a/pkg/events/core.go#L11502-L11533) - be aware of possible changes to the definition linked above, so always check the main branch.

Or the user can test the event output in CLI before defining a policy, e.g:
```

Reply:
Right below we have the output of the command which still have "args" on it. Could you run the command and check the changes? If alright, copy and paste it replacing the json output.

Reply:
Tks to spot it @geyslan, as synced with @yanivagman this change will be handled by the new event structure @josedonizetti